app.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
**TODO FOR LOST POINTS**

This is a top-level comment to remind you that you need your ``eslintrc`` and a ``gitignore`` in the root of your file per the instructions.